evergreen/staticfiles/css/styles.css

```diff
@@ -79,13 +83,17 @@ footer {
 
 .post_form {
     background-color: #b9efbd;
+
     height: 15%; 
+
 }
 
 .feed_posts {
     display: inline;
     align-content: center;
+
     padding: 5%
```

Comment:
you dropped the semicolon here. every other change in this file is unnecessary whitespace that i think was better before